inc/zoo/swar/associative_iteration.h

```diff
@@ -483,6 +493,82 @@ constexpr auto exponentiation_OverflowUnsafe_SpecificBitCount(
     );
 }
 
+using S = SWAR<4, u16>;
+
+/** Transforms a number into a number into a binary tally.
+ * E.g. 0b0011 (3) -> 0b0111
+ * It seems that trying to get the lane width as a tally is weird and overflowy.
+ * */
+template <typename S>
+constexpr auto base2TallyTransform_Plural(S input) {
+    constexpr auto two = S{meta::BitmaskMaker<typename S::type, 2, S::NBits>::value};
+    constexpr auto one = S::LeastSignificantBit;
+    typename S::type v = exponentiation_OverflowUnsafe_SpecificBitCount<S::NBits>(two, input).value() - one;
+    return S{v};
+}
+static_assert(base2TallyTransform_Plural(S{0b0001'0010'0011'0011}).value() == 0b0001'0011'0111'0111);
+static_assert(base2TallyTransform_Plural(S{0b0000'0001'0010'0011}).value() == 0b0000'0001'0011'0111);
+static_assert(base2TallyTransform_Plural(S{0b0100'0001'0010'0011}).value() == 0b1111'0001'0011'0111);
+static_assert(base2TallyTransform_Plural(S{0b0000'0000'0000'0001}).value() == 0b0000'0000'0000'0001);
+static_assert(base2TallyTransform_Plural(SWAR<8, uint16_t>{0b000000111'00000101}).value() == 0b01111111'00011111); // 7 -> 5
```

Comment:
Use whitespace to make the point of the comparison/test evident!
You not only trample the whitespace of others, but fail to use it